### PR TITLE
revising jscsrc

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

File: linters/.jscsrc [hunk 1]
```diff
@@ -60,7 +59,7 @@
   "requireSpacesInsideObjectBrackets": {
     "allExcept": ["}"]
   },
-  "safeContextKeyword": ["that", "self"],
```
Comment (gvn) [inline linters/.jscsrc]: I'm pretty sure `that = this` is Crockford either trolling or being "clever".

File: linters/.jscsrc [hunk 2]
```diff
@@ -23,10 +23,9 @@
   "disallowTrailingComma": true,
   "disallowTrailingWhitespace": true,
   "disallowYodaConditions": true,
-  "maximumLineLength": 80,
+  "maximumLineLength": 120,
   "requireBlocksOnNewline": 1,
   "requireCamelCaseOrUpperCaseIdentifiers": true,
-  "requireCapitalizedComments": true,
```
Comment (gvn) [inline linters/.jscsrc]: I'm not sure I get the reasoning behind this one...

Comment (simonwex) [inline linters/.jscsrc]: The reasoning is easy for people who think this is funny: Bad grammar makes me [sic]. I'm one of those people, but I'm allergic to it without the `exceptMultiline` value.

Comment (gvn) [inline linters/.jscsrc]: Yeah, I'm fine with it, but it does seem a *tad* pedantic. :wink:

Comment (simonwex) [inline linters/.jscsrc]: It's *super* pedantic, keep it out.

Comment (gvn) [inline linters/.jscsrc]: LOL. Will do...

**Pull request merged**